README.md

```diff
@@ -0,0 +1,27 @@
+# RideMetricSampleApp
+This is a sample app uses RideMetric SDK and can be expanded to include specific functionality and features.
```

Comment:
I think it will be great if we generate Javadoc documentation for the code. It will solves the SDK's documentation problem.